doc/contribute/project_roles.rst

```diff
@@ -3,16 +3,16 @@
 TSC Project Roles
 *****************
 
-Overview
-########
+Main Roles
+##########
 
 TSC projects generally will involve *Maintainers*, *Collaborators*, and
 *Contributors*:
 
 **Maintainer**: lead Collaborators on an area identified by the TSC (e.g.
 Architecture, code subsystems, etc.). Maintainers shall also serve as the
-area’s representative on the TSC as needed.  Maintainers may become voting
-members of the TSC by approval of the existing TSC voting members.
+area’s representative on the TSC as needed. Maintainers may become voting
+members of the TSC under the guidelines stated in the project Charter.
```

Comment:
@kestewart could you check this amendment - whether it is fine for you?

Reply:
We'd need to update the charter to be explicit here.    Can you help me understand the background on why the existing TSC voting members as a statement isn't felt to be sufficient?

Reply:
The rationale behind this minor change is the following: If TSC should be allowed to add additional Maintainers to the committee, this needs to be done in a way that it is compatible with the Charter, which currently states that "community members may become TSC voting members at a rate no more than 1 per quarter".

This rule applies now, but may be changed (or even removed) in the future. Therefore, what I wanted here is to state - in a generic way- that TSC should be allowed to add new members in the committee as long as this is done in the way the charter specifies it.